### ISSUE TITLE
Test with everdrive64 X7

### ISSUE THREAD

**Issue by anacierdem** [anacierdem/ed64#6]
I don't know if it is register compatible.

**Comment by lambertjamesd** [comment]
I just got an X7 and am testing it out. It doesn't seem to work. I am doing some digging to see if I can figure it out.

**Comment by anacierdem** [comment]
The working mechanism is different from what I see of the new source code. OS 3.04 seem to bring support for v3 (http://krikzz.com/pub/support/everdrive-64/x-series/OS/changelist.txt) I will verify if it is working for v3 cartridges and we can start a new major version compatible with the new OS. We can then add x7 support mimicking its behavior. As I don't have an x7 this may take a while until stable though.

**Comment by networkfusion** [comment]
From a USB point of view, the V3 and X7 are the same with the new X-Series OS. The only difference is the result of hardware detection. As such, you will be fine developing the solution on the V3 cart with the latest X-Series OS.

**Comment by anacierdem** [comment]
Yes I have ween that. Now the only thing I need is making this 3.04 compatible when I have the time.

**Comment by networkfusion** [comment]
Yeah, the main difference I have found when working on it (to bring the same X-Series USB interface to My unofficial OS is that the USB buffer is now able to be variable rather than a static 512 buffer (which makes it difficult to keep backwards compatibility for the loader on the 2.12.x OS).
BTW, latest OS firmware is 3.05.

**Comment by anacierdem** [comment]
@lambertjamesd can you give it another try when you have the time? It should work with the latest changes.
